### PR TITLE
[IOTDB-4497] Improve NodeStatus definition

### PR DESCRIPTION
**Pull request by CRZbulabula**
Add more detailed NodeStatus definition on user docs.

### CODE REVIEW

File: docs/UserGuide/Cluster/Cluster-Maintenance.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ The ConfigNode statuses are defined as follows:
 
 - **Running**: The ConfigNode is running properly.
 - **Unknown**: The ConfigNode doesn't report heartbeat properly.
+  - Can't receive data synchronized from other ConfigNodes
```
Comment (Beyyes) [inline docs/UserGuide/Cluster/Cluster-Maintenance.md:85]: synchronized is adj. or v.

File: docs/zh/UserGuide/Cluster/Cluster-Maintenance.md [hunk 2]
```diff
@@ -130,8 +132,13 @@ DataNode 的状态机如下图所示：
 
 - **Running**: DataNode 正常运行，可读可写
 - **Unknown**: DataNode 未正常上报心跳，ConfigNode 认为该 DataNode 不可读写
+  - 少数 DataNode Unknown 不影响集群读写
 - **Removing**: DataNode 正在移出集群，不可读写
+  - 少数 DataNode Removing 不影响集群读写 
 - **ReadOnly**: DataNode 磁盘剩余空间低于 disk_warning_threshold（默认 5%），DataNode 可读但不能写入，不能同步数据
+  - 少数 DataNode ReadOnly 不影响集群读写
+  - ReadOnly 状态下可以删除元数据和数据，可以删除 Database
+  - 所有 DataNode 处于 ReadOnly 状态时，集群不可写入元数据和数据，仍可以创建 Database
```
Comment (Beyyes) [inline docs/zh/UserGuide/Cluster/Cluster-Maintenance.md:141]: 是否可删除数据？
能否列个表格注明各种情况下是否可读、写、删数据

Comment (CRZbulabula) [reply docs/zh/UserGuide/Cluster/Cluster-Maintenance.md:141]: Added~

File: docs/UserGuide/Cluster/Cluster-Maintenance.md (hunk 1 above)
Comment (CRZbulabula) [reply docs/UserGuide/Cluster/Cluster-Maintenance.md:85]: This is an attributive clause that leaves out that~

**Review by Beyyes — APPROVED** [review]
LGTM

**Pull request merged**